xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -484,6 +485,32 @@ final class DatabaseService {
         .first
     }
     
+    func loadUserProfile(did: Did) throws -> Data? {
```

Comment:
Let's add a test for this

Reply:
Added `Tests_UserProfileService.testReadUserProfile` and renamed this method to `readUserProfile` for consistency with other methods in the file.